i18n/en.json

```diff
@@ -936,6 +936,10 @@
     "translation": "Already deleted id={{.PostId}}"
   },
   {
+    "id": "api.post.update_post.system_message.app_error",
+    "translation": "Unable to update system message"
```

Comment:
I was going off some of the next few error messages like "Unable to get post", "Unable to decode preferences", etc. We're incredibly inconsistent with error messages, but I don't think it matters with this one since users shouldn't see it if the app is working correctly